src/Microsoft.ServiceFabric.Services.Remoting/V2/Runtime/ExceptionConvertorHelper.cs

```diff
@@ -0,0 +1,165 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.ServiceFabric.Services.Remoting.V2.Runtime
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Fabric;
+    using System.IO;
+    using System.Runtime.Serialization;
+    using System.Xml;
+    using Microsoft.ServiceFabric.Services.Communication;
+    using Microsoft.ServiceFabric.Services.Remoting.FabricTransport.Runtime;
+
+    internal class ExceptionConvertorHelper
```

Comment:
I feel a 'helper' class should not be one of our major classes. Can we change the name to _ExceptionConversionHandler/ExceptionRemotingHandler_

Same for this class on the client side as well

Reply:
Changed the class name to ExceptionConversionHandler.